doc/user_guide/environment_variables.md

```diff
@@ -52,14 +52,18 @@ DLite-specific environment variables
   - **DLITE_ATEXIT_FREE**: Free memory at exit.  This might be useful to avoid
     getting false positive when tracking down memory leaks with tools like valgrind.
 
-  - **DLITE_BEHAVIOR_<name>**: Enables/disables the behavior `<name>`.
+  - **DLITE_BEHAVIOR**: Enables/disables the all behaviors by default.
```

Comment:
No, not al behaviours. Must be rewritten. This is only for behaviour changes, right?

Reply:
This option actually turns on/off all behaviors by default. It used for testing (assuming that the behavior changes are independent).